### ISSUE TITLE
AttributeError: module 'torchvision.models' has no attribute 'detection'

### ISSUE THREAD

**Issue by ksivaman** [pytorch/pytorch#23236]
## 🐛 Bug

Detection module is not present in torchvision.models

## To Reproduce

1. import detection from torchvision,models in a python script

## Environment

Please copy and paste the output from our
[environment collection script](https://raw.githubusercontent.com/pytorch/pytorch/master/torch/utils/collect_env.py)
(or fill out the checklist below manually).

You can get the script and run it with:
```
wget https://raw.githubusercontent.com/pytorch/pytorch/master/torch/utils/collect_env.py
# For security purposes, please check the contents of collect_env.py before running it.
python collect_env.py
```

 - PyTorch Version (e.g., 1.0):
 - OS (e.g., Linux):
 - How you installed PyTorch (`conda`, `pip`, source):
 - Build command you used (if compiling from source):
 - Python version:
 - CUDA/cuDNN version:
 - GPU models and configuration:
 - Any other relevant information:

**Comment by zhangguanheng66** [comment]
If this is an issue related to torchvision, you should bring it up here.
https://github.com/pytorch/vision/issues

**Comment by crazysal** [comment]
Faced the same, realized it is a brand new feature only available in lts 1.1.0 not before that...
